Content: New alien species: Incipias (no.3) (#8949)
endless-sky/endless-sky: data/map planets.txt

		threshold 8000
		fleet Quarg 2

planet Humo
	attributes "requires: gaslining"
	landscape land/humo

> 	description `Interestingly this gas giant is not only made from gaseus elements, sand is being pushed around in the air and cumulates on the windows of your ship. A constant updraft from the center of the planet seems to be the reason for sand to exist in the upper atmosphere. While the sand itself is not dangerous there is the risk of it getting into your ships mechanics.`

Reviewer: ```suggestion
	description `Interestingly this gas giant is not only made from gaseous elements, sand is being pushed around in the air and cumulates on the windows of your ship. A constant updraft from the center of the planet seems to be the reason for sand to exist in the upper atmosphere. While the sand itself is not dangerous there is the risk of it getting into your ships mechanics.`
```

> 	description `Interestingly this gas giant is not only made from gaseous elements, sand is being pushed around in the air and cumulates on the windows of your ship. A constant updraft from the center of the planet seems to be the reason for sand to exist in the upper atmosphere. While the sand itself is not dangerous there is the risk of it getting into your ships mechanics.`

Reviewer: ```suggestion
	description `Interestingly, this gas giant is not only made from gaseous elements; sand is being pushed around in the air and cumulates on the windows of your ship. A constant updraft from the center of the planet seems to be the reason for getting the sand into the upper atmosphere. While the sand itself is not dangerous, there is the risk of it getting into your ships mechanics.`
```

		threshold 2000
		fleet "Large Militia" 7

planet "Pon'tes"
	government "Hicemus"
	landscape land/station16
	"required reputation" 10
	outfitter "Hicemus"
	shipyard "Hicemus"
	description "Pon'tes is uncomfortable, even by the standards of a station. As the Incipias are able to propel themselves through the air, there are no handles or surfaces for you to grip onto. The simulated gravity caused by the rotation of the station is not helping."

> 	description "The station is a bit larger than the most human ones and a lot of Incipias are living here. They all seem very busy as they float through the tube-like hallways."

Author: is the station large because it has a high population or does it have a high population because it is large?

> 	description "The station is a bit larger than the most human ones and a lot of Incipias are living here. They all seem very busy as they float through the tube-like hallways."

Author: cool like it

	spaceport "The station has no normal spaceport - only a small docking area. Most of the people present appear to be workers, as opposed to visiting captains."

> 	spaceport "Most of the ships are delivering minerals but some also deliver supplies from Redias. Only a few seem to transport people."

Reviewer: Didn't they design transport crafts specifically  because of Pon'tes?

Author: That does not mean there have to be a lot of them, right?

Author: ```suggestion
	spaceport "Most of the ships are delivering minerals but some also deliver supplies from Redias. In an small area separated from the rest there is a constant buzzing of small transport ships transporting civilians in both directions."
```

		threshold 3000
		fleet "Large Militia" 8

planet "Redias"
	landscape land/redias
	attributes "requires: gaslining"
	"required reputation" 10
	description "The home planet of the Incipias is reigned over by powerful storms. Lightning and thunder appear without pause and a powerful wind moves the clouds at a disorienting speed."

> 	description "The Incipias have adjusted perfectly to this harsh environment. Their cities are built like them: large balloon-like structures which hold spheres of buildings in the air. The Incipias themselves can move freely through the atmosphere, so the cities have no roads. To travel larger distances, they use gliders that can fly against the powerful winds."

Author: it is supposed to mean, spheres consisting of buildings

	bribe 0
	security 0

planet "Spec Inci"
	attributes "incipias quarg" quarg
	landscape land/sky7

> 	description "This small moon seems to be used as just a shipment and processing base. Ships are dropping off large amounts of raw ore that then get processed by machines. You wouldn't have even noticed them if it were not for their large size, as they are surprisingly silent when working. Other ships load the processed materials and fly away."

Author: really like that change

> 	spaceport "The spaceport is fully automated. You can see no Quarg that is not part of a ship's crew. When a ship lands on one of the landing pads on the surface an elevator moves it underground into one of the many hangars. One section of the ship is restricted and a large amount of military ships can be seen."

Reviewer: ```suggestion
	spaceport "The spaceport is fully automated. You can see no Quarg that is not part of a ship's crew. When a ship lands on one of the landing pads on the surface an elevator moves it underground into one of the many hangars. One section of the spaceport is restricted and a large amount of military ships can be seen."
```

	bribe 0
	security 0

planet "Spec Inci"
	attributes "incipias quarg" quarg
	landscape land/sky7
	description "This small moon seems to be used mainly as a shipment and processing base, run with the typical efficiency of the Quarg. You wouldn't have even noticed the freighters dropping off large amounts of raw ore for processing if it were not for their large size, as they are surprisingly silent when working. Other ships load the processed materials and fly away."

> 	spaceport "The spaceport is fully automated, and you can see no Quarg that is not part of a ship's crew. When a ship lands on one of the landing pads on the surface an elevator moves it underground into one of the moon's many hangars. One section of the spaceport is restricted, where a large amount of military ships can be seen."

Author: I thought of a very big cavern

> 	spaceport "The spaceport is fully automated, and you can see no Quarg that is not part of a ship's crew. When a ship lands on one of the landing pads on the surface an elevator moves it underground into one of the moon's many hangars. One section of the spaceport is restricted, where a large amount of military ships can be seen."

Reviewer 2: What about that spaceport news item where two Quarg crews switch ships?

> 	spaceport "The spaceport is fully automated, and you can see no Quarg that is not part of a ship's crew. When a ship lands on one of the landing pads on the surface an elevator moves it underground into one of the moon's many hangars. One section of the spaceport is restricted, where a large amount of military ships can be seen."

Reviewer 3: Hmm, fair point. Perhaps the Quarg would have their civilian and military hangars separated from each other, but not have their ships completely hidden from view:
```suggestion
	spaceport "The spaceport is fully automated, and you can see no Quarg that is not part of a ship's crew. When a ship lands on one of the landing pads on the surface an elevator moves it underground into one of the moon's many hangars. One section of the spaceport is restricted, where a large amount of warships are being lowered into what is presumably a Quarg military base."
```

	government Uninhabited
	security 0

planet Treser
	attributes "requires: gaslining"
	landscape land/treser

> 	description `A dark world lies beneath you. Some chemical compound in the clouds is absorbing over ninety percent of the incoming light. This results in a enormous amount of heat in the atmosphere of this gas giant. The extreme temperatures are so taxing on your ship that you are barely able to fly through the uppermost layers. When the heat exceeds a certain point the gas becomes plasma, creating small spots of light on this world of eternal darkness.`

Reviewer 3: ```suggestion
	description `A dark world lies beneath you. Some chemical compound in the clouds is absorbing over ninety percent of the incoming light. This results in an enormous amount of heat in the atmosphere of this gas giant. The extreme temperatures are so taxing on your ship that you are barely able to fly through the uppermost layers. When the heat exceeds a certain point the gas becomes plasma, creating small spots of light on this world of eternal darkness.`
```

		threshold 2000
		fleet "Large Militia" 8

planet "Ut Divitas"
	attributes "incipias quarg" quarg
	landscape land/canyon13
	description "The surface is bare and there is no sign of life to be seen anywhere. A few lonely but giant cities sprawl like flowers on the surface of the moon, the architecture is abstract and elegant like nothing humans ever build."

> 	spaceport "The spaceport is a small one, consisting of only a few landing pads. Ships that are landed here are mostly civilian. The Quarg working there seem to be confused by your presence."

Author: noted

		threshold 2000
		fleet "Large Militia" 8

planet "Ut Divitas"
	attributes "incipias quarg" quarg
	landscape land/canyon13

> 	description "The surface is bare and there is no sign of life to be seen anywhere. A few lonely but giant cities sprawl like flowers on the surface of the moon, and the architecture is abstract and elegant like nothing humans have ever built."

Reviewer 3: Why would there be no sign of life anywhere, if there are Quarg working in the spaceport?
```suggestion
	description "The surface here is bare and there is no sign of life to be seen anywhere. A few lonely but giant cities sprawl like flowers on the surface of the moon, and the architecture is abstract and elegant unlike anything humans have ever built."
```

		threshold 2000
		fleet "Large Militia" 8

planet "Ut Divitas"
	attributes "incipias quarg" quarg
	landscape land/canyon13

> 	description "The surface here is bare and there is no sign of life to be seen anywhere. A few lonely but giant cities sprawl like flowers on the surface of the moon, and the architecture is abstract and elegant unlike anything humans have ever built."

Reviewer 3: ```suggestion
	description "The surface here is bare and there is no sign of life to be seen anywhere. A few lonely but giant cities sprawl like flowers on the surface of the moon, and the architecture is abstract, elegant, and unlike anything humans have ever built."
```

> 	spaceport "The spaceport is a small one, consisting of only a few landing pads. Ships parked here are mostly civilian. The Quarg working here seem to be confused by your presence."

Reviewer 3: ```suggestion
	spaceport "The spaceport is a small one, consisting of only a few landing pads. Ships parked here appear to be mostly civilian. The Quarg working here seem to be confused by your presence."
```

	description `	The life-bearing layer only extends downward a few hundred kilometers before pressures rise and the temperature becomes inhospitable to life. Deeper down, the atmosphere is barren and lifeless.`
	government Uninhabited

planet Vulpa
	attributes "requires: gaslining"
	landscape land/vulpa

> 	description `At first sight this looks like a friendly gas giant. The blue tones invoke the feeling of an ocean world, but looks can be deceiving. The clouds contain countless small glass droplets moving at tremendous speeds, cutting through everything in their way. Your spaceship's hull is able to deflect most of them but it is advisable not to stay here for too long.`

Author: whoops gliese should have a different description, will fix
